client/src/components/VideosList.jsx

```diff
@@ -0,0 +1,105 @@
+import React, { useEffect, useState } from "react";
+import Video from "./Video";
+import AddVideo from "./AddVideo";
+import "../index.css";
+import Search from "./Search";
+
+function VideosList() {
+  const [videos, setVideos] = useState([]);
+  const [loading, setLoading] = useState(true);
+  const [searchResult, setSearchResult] = useState([]);
+
+  useEffect(() => {
+    fetch("https://node-server-full-stack.onrender.com/videos")
```

Comment:
Use an environment variable for this API url - don't just hardcode the string. Do you understand why this is important?

Reply:
I chaned this